lib/internal/test_runner/test.js

```diff
@@ -549,14 +573,16 @@ class Test extends AsyncResource {
       }
     });
 
+    let testTimeout;
+
     try {
       if (this.parent?.hooks.before.length > 0) {
         await this.parent.runHook('before', this.parent.getRunArgs());
       }
       if (this.parent?.hooks.beforeEach.length > 0) {
         await this.parent.runHook('beforeEach', { args, ctx });
       }
-      const stopPromise = stopTest(this.timeout, this.signal);
+      testTimeout = stopTest(this.timeout, this.signal);
```

Comment:
this name is misleading, it is either timeout, or abort signal aborted, that is why it was called `stopPromise`

Reply:
restore to `stopPromise` but the `stopTest` result is not a promise